src/transactions/reducer.ts

```diff
@@ -137,6 +156,15 @@ export const reducer = (
 export const standbyTransactionsSelector = (state: RootState) =>
   state.transactions.standbyTransactions
 
+export const watchablePendingTransactionSelector = createSelector(
+  [standbyTransactionsSelector],
```

Comment:
Sorry, I don't want to use `pendingStandbyTransactionsSelector` because it fills the `transactionHash` with `''`.
I'm gonna create a new type for the "watchable" transactions.